packages/widgetbook_cli/bin/models/project.dart

```diff
@@ -0,0 +1,32 @@
+import '../commands/helpers/helpers.dart';
+
+class Project {
```

Comment:
This should be a freezed class. 
The name is suboptimal because we also have a `Project` in Widgetbook Cloud which will cause naming conflicts in the future. `Package` or `DartPackage` might be a lot better.